app/notifications/rest.py

```diff
@@ -88,13 +88,18 @@ def get_all_notifications():
 @notifications.route('/notifications/<string:notification_type>', methods=['POST'])
```

Comment:
this is the old v1 public endpoint - we shouldn't be touching this cos it needs an api key and everything

Instead, the flow should use this function (which is accessed via the route `/<uuid:service_id>/send-notification`):

https://github.com/alphagov/notifications-api/blob/master/app/service/send_notification.py#L37